<launch>
    <include file="$(find turtlebot3_gazebo)/launch/turtlebot3_empty_world.launch" />
    <node name="pose2d_publisher" pkg="turtle_controller" type="odom2pose.py"/>
    <node name="turtle_control_server" pkg="turtle_controller" type="controller_server" output="screen"/>
</launch>